File: NetFlexAPI/NetFlexAPI/Models/UserSubscription.cs
using System.ComponentModel.DataAnnotations;

namespace NetFlexAPI.Models;

public class UserSubscription
{
    [Key]
    public Guid UserId { get; set; }
    public Guid SubscriptionId { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime FinishDate { get; set; }
}

File: NetFlexAPI/GiraffeAPI/Models/UserSubscription.fs
module GiraffeAPI.Models.UserSubscription

open System
open System.ComponentModel.DataAnnotations
open Microsoft.EntityFrameworkCore.Metadata.Internal
open Ply.TplPrimitives

[<CLIMutable>]  
type UserSubscription =
    {
        [<Key>]
        UserId : Guid
        SubscriptionId : Guid
        StartDate : DateTime
        FinishDate : DateTime
    }
    
    